src/utils/bridge.js

```diff
@@ -7,40 +7,53 @@ import apiFetch from '@wordpress/api-fetch';
  * Internal dependencies
  */
 import parseException from './exception-parser';
+import { debug } from './logger';
+import { isDevMode } from './dev-mode';
 
 /**
- * Notifies the native host that the editor has loaded.
+ * Generic function to dispatch messages to both Android and iOS bridges.
+ *
+ * @param {string} methodName The name of the method to call on the native side.
+ * @param {Object} args       Arguments object to pass to the method.
  *
  * @return {void}
  */
-export function editorLoaded() {
+function dispatchToBridge( methodName, args = {} ) {
+	debug( `Bridge event: ${ methodName }`, args );
+
+	// Android bridge - extract values in property definition order
 	if ( window.editorDelegate ) {
-		window.editorDelegate.onEditorLoaded();
+		const method = window.editorDelegate[ methodName ];
+		if ( method ) {
+			method.apply( window.editorDelegate, Object.values( args ) );
+		}
 	}
 
+	// iOS webkit bridge - use arguments object directly
 	if ( window.webkit ) {
 		window.webkit.messageHandlers.editorDelegate.postMessage( {
-			message: 'onEditorLoaded',
-			body: {},
+			message: methodName,
+			body: args,
 		} );
 	}
 }
```

Comment:
Introduced an event dispatcher utility to reduce repetition for common platform logic, and added debug logs for bridge events.

Reply:
Noting that I smoke tested the editor on both iOS and Android to avoid missing a regression from these bridge changes. I tested both development server and builds, both the local and remote editor.